### DISCUSSION TITLE
Participatory approaches and methods for communities

### DISCUSSION THREAD

**Discussion by Liyubov** [life-itself/community#345] [Research]
Hello everyone,

I am posting here several articles on participatory methods and participatory approaches for communities.

As mentioned this week maybe it makes sense to stay on this topic, and happy to facilitate discussions on this 
this Friday. 

Some papers to look at and discuss are: 

- one is related to what we have been touching in discussions last time on participatory rural approaches with great overview of methods (actual even now) https://www.sciencedirect.com/science/article/abs/pii/0305750X94901414
- recent paper by Utrecht University colleagues: [Power and politics in stakeholder engagement: farm dweller (in)visibility and conversions to game farming in South Africa](https://www.ecologyandsociety.org/vol23/iss3/art32/)
- [co-productive agility and four collaborative pathways to sustainability transformations](https://www.sciencedirect.com/science/article/pii/S0959378021002016?via%3Dihub)
- materials on current work on collaborations with communities by American Geophysical society [(AGU)](https://thrivingearthexchange.org/)

Happy to hear more from you what are relevant resources and papers / overviews, existing practices, for urban projects and beyond.

P.S.: Talk on citizen assembly in non democratic setups on 9th March 6pm CET: 
https://docs.google.com/forms/d/e/1FAIpQLSdmHGDucHKcV3iKhXaqXq4tzZ9chBCkPJ71CVJLN2oI2MK8QQ/viewform

**Comment by rufuspollock** [comment]
Amazing @Liyubov. Thank-you for posting 🙂

One small thing is that some part of the post seems duplicated and perhaps you could add a bit more info to the title for others (ie. participatory approaches to ... *what* 🙂)

**Reply by rufuspollock** [reply]
Update: i've updated the title and formatted the text a bit 😄

**Comment by Liyubov** [comment]
Hey, 
just edited it and added some notes and points to discuss which also were pointed out by John:

- Participatory methods applied to urban issues (but actually to any X issues)

- The main challenge appeared to be that some stakeholders are not recognized as stakeholders
  Invisibility is linked to specific institutional and personal relations (Chambers 1994) https://www.sciencedirect.com/science/article/abs/pii/0305750X94901414
- Stakeholders mapping tools: working with NGOs and balancing routine reporting work with creative research work in trans-un-disciplinary 
  participatory projects - best practices and failures (Spielberg et al. 2021) [https://www.ecologyandsociety.org/vol23/iss3/art32/](https://www.google.com/url?q=https://www.ecologyandsociety.org/vol23/iss3/art32/&sa=D&source=docs&ust=1677510828178435&usg=AOvVaw2iqVh3RSZG8JttSTmqrtTG)

**Reply by rufuspollock** [reply]
> The main challenge appeared to be that some stakeholders are not recognized as stakeholders. Invisibility is linked to specific institutional and personal relations (Chambers 1994)

@Liyubov 👍  as often the issue here will be the politics rather than the knowing what to do (i.e. what process we *should* do). There is a reason people are not recognized as stakeholders ...

**Reply by baouroux** [reply]
You raise my next personal challenge Rufus....how can I get more involved in the sphere of politics?

Something I've totally avoided over all these years, yet fascinated by authors such as John Rawls. And, I guess, the fundamental question for me is (taking you comment above to one more iteration) is there a way of "_knowing what to do / what we should do_" with regards the polticial systems themselves?

I find myself being drawn to adopting the lens of an anthropolgist, how to make sense of the ways politics has evolved or not over generations, as my main starting point. We have very good friends living near us, who have moved from South Africa, Iran and Colombia....the first-hand stories over our lunches make me really question how I can engage, relate-to or possibly 'make sense' of these political situations, and corruption. 

PS An incredible theatrical window into the complexity of politics I came across recently here:  https://www.wrestlingjerusalem.com/

**Reply by rufuspollock** [reply]
Well, our big vision at Life Itself is to contribute to the creation of a mass political movement over the long-term and get some folks elected 😄  https://lifeitself.org/tao/scqh#hypothesis

> Within our lifetime and six generations hence, we will develop a federation of resilient, flourishing communities based on a shared culture (views, values and practices including the possibility of transformation, the primacy of (inter)being and getting stuff done). These communities will engage with broader society and ally with like-minded groups to foster a culture of wisdom and awakening 2 that can transform society and will be committed to practical action, including large scale political efforts to address collective challenges.

**Reply by asimong** [reply]
what would be the pros and cons of working with DIEM25? would it be possible e.g. to introduce some excellent practice to them and transform their potential?

**Comment by baouroux** [comment]
Thanks for these references Liubov, diverse and inspiring. 

Picking up on stakeholder mapping, this is an extract from a Dave Snowden talk and the application of the "self-signification" of narratives. Although it is an organisational design case study (and he is actually talking at an UX user design conference!), there is a direct possible for citizen applications.

These narratives maps are exactly what we delivered for our organisational development, employee culture, engagement and leadership projects. The principles were to disintermediate the external consultants in the ownership and actionability of the data - by having the co-workers access themselves the data and have smaller working groups explore 'how do we get more stories like these, less like those?', and propose actions in highly granular, and context specific terms: 

[SenseMaker Data Landscapes -  Complexity Informed Design Thinking - Dave Snowden](https://youtu.be/mzFjoTqIfCA) at Lean WX 2015 

Happy to forward if of interest some the www.cynefin.io SenseMaker case studies in the NGO aid projects space.

**Comment by Liyubov** [comment]
Just wanted to introduce @mattosborn https://github.com/mattosborn who would be really great to discuss here. 
We do have several threads of discussions here on different research topics.

**Reply by mattosborn** [reply]
Thank you @Liyubov ! I'm slowly absorbing and digesting and finding my way around. Loving all the topics here and very keen to get involved in some way.

**Comment by baouroux** [comment]
Following up on a question from Liubov on are any data (like docs or scans) from sensemaking narratives cocreation sessions with citizens e.g. using post it notes etc. anywhere available?", herewith a couple of pointers specifically relating to to citizen projects. 
I have taken the references from the publically available data from the original developers of the approach The Cynefin Center and their tool SenseMaker. My own client projects data sources using SenseMaker are now offline, and the reports contain confidential client narratives, so best I refer you to the Cynefin Center's work.
I'm afraid these are static reports vs. accessing the data through a live dashboard - if of interest, I can ask a previous colleague if there are some client data projects publically available.

- “My Malmö” is a a collaborative initiative established by the NGO Forward Malmö using narratives and The Cynefin Center's SenseMaker software. [A report is here with some data extracts](https://drive.google.com/file/d/1XHSjC0NGTT5aW1wCUYQUdBMoExPmwadZ/view?usp=sharing).
- Finland Sitra: https://www.sitra.fi/en/articles/sensemaker-tool-decision-making-new-kind-world/

Plus the concept document of [using SenseMaker / narratives sensemaking in citizen projects](https://drive.google.com/file/d/1XJtBW2mfDBan0BOjjOjQiV7OaHjMpuHU/view?usp=sharing), and [these citizen engagement case studies](https://drive.google.com/file/d/1XTEXmBG7Ts4kfLTNqX0sYxr0qbrmzPKp/view?usp=sharing) with SenseMaker.

While looking at the Malmo case study, I came across [this other paper mentioning the "Malmo Living Labs" project](https://drive.google.com/file/d/1X4uW-wvxjJlyVrKFYY6Ocf7koHkAqEcF/view?usp=sharing), and thought this an interesting overview of citizen engagement. I'd be interest to hear what you think of the paper!

**Comment by Liyubov** [comment]
Thanks a lot, @baouroux , really relevant and just going through the literature now. 
My Malmoe  initiative looks very interesting. 

I also know some of the work UNDP https://medium.com/@undp.innovation/  was doing along this line, some articles on their medium blogs, but not so much open data which I found there on this. We can maybe discuss it more the next time we synchronise with each other. 

Thank you.

**Comment by baouroux** [comment]
Just picking up on a couple of exchanges with @Liyubov and @msantolini on sensemaking, narratives, and more recently on the possible connection between big data and sensemaking.

Thanks Liubov for sharing this paper reference: https://anthrosource.onlinelibrary.wiley.com/doi/epdf/10.1111/1559-8918.2018.01203"Human Sensemaking in the Smart City: A Research Approach Merging Big and Thick Data" Vrije Universiteit Brussel

"Thick Data" is the alternative term to qualitative inputs (e.g. narratives, focus groups, experiential feedback), or otherwise Nora Bateson uses "Warm Data".

Integrating both "Thick/Warm" with "Big" (or Quant) is super interesting for my own imagination as to where the narratives sensemaking future innovations lie, at both the macro level (e.g. Narrative Economics by Robert Schiller), to the micro, more day-to-day applications such as the above paper describes in analysing the experience of cycle travel in urban areas.

I wanted to share with you a very recent project proposal that I am working on, that connects to exactly the same principles of correlating Quant data with narratives. It's in the frontier scene of the Institute of Noetic Sciences (IONS) and Princeton Engineering Anamolies Research Lab (PEAR) projects, and how Random Event Generators pick up human emotional experiences: https://gowyrd.org/research/

Wyrd Tech have imported the entire PEAR Lab instruments and archives to the UK, and I have already collaborated with them for some filming work. They are raising research and development grants for the next generation of applications - in particular Random Event Generators that pick-up on human emotions in the 'field'. I am in the process of drafting this [proposal to add the sensemaking / Qual / thick data dimension here:](https://docs.google.com/document/d/18O8WmS70TujbXP143p5Z8xfgOcvv1j09z-TIzT45sjc/edit?usp=sharing)

Any synergies in our projects would be great to discuss!

**Reply by Liyubov** [reply]
thanks John for following up. Would be great to see who are actually the main stakeholders in this area. 
I saw many initiatives like Maelmo and Sitra in Finland, who work on sensemaking and mapping, do share if you have more labs or groups working on it. 
thank you!

**Comment by msantolini** [comment]
@baouroux, @laurenwigmore @nathenf had a great idea - to put QR codes around the Hub that link to google forms where one can say what rooms they are in and answer a simple question. I think it's a great opportunity to prototype some sensemaking here! We already have a [questionnaire](https://forms.gle/D7kpLz8hNVEPdGri8), but I think we could stick to one question only. For kitchen/dinner room I would use the question on food. For other rooms, my gut feeling would be to ask something about the practice and experience in free text, and then maybe use 2 scales (in the same way you showed before), that could be related to how alone vs collective the moment is, and how inner (mediation etc) vs outer (e.g. hackathon) it is. Would you have idea on how to design this best? I believe you are the expert on best way to make sense of these experiences :)

**Reply by msantolini** [reply]
Created a specific discussion thread for this in #691

**Comment by Liyubov** [comment]
Wanted to bring attention to the materials from the Centre of Constructive Communication from MIT are articles from Dimitra 
https://www.ccc.mit.edu/publications/

Brandon mentioned to us this material with Marc. Maybe you know about this work @baouroux and all...
